File: comms/bulk-messaging/messages/fetch-message/fetch-message.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "TWILIO_ACCOUNT_SID",
        authToken: "TWILIO_AUTH_TOKEN",
    });
    await client.messages.fetch("comms_message_01h9krwprkeee8fzqspvwy6nq8");
}
main();

File: comms/bulk-messaging/messages/fetch-message/fetch-message.go
package example

import (
    context "context"

    client "github.com/twilio/twilio-comms-go/twilio/client"
    option "github.com/twilio/twilio-comms-go/twilio/option"
)

func do() {
    client := client.NewWithOptions(
        option.WithBasicAuth(
            "<username>",
            "<password>",
        ),
    )
    client.Messages.Fetch(
        context.TODO(),
        "comms_message_01h9krwprkeee8fzqspvwy6nq8",
    )
}
